src/main/java/org/mineacademy/boss/model/BossPlaceholders.java

```diff
@@ -204,14 +207,23 @@ else if ("respawn".equals(secondArg)) {
 			final Location location = hasBoss ? closestBoss.getEntity().getLocation() : null;
 			final TreeMap<Double, Player> lastDamageMap = hasBoss ? closestBoss.getBoss().getRecentDamagerPlayer(closestBoss.getEntity(), false) : null;
 
-			double lastDamage = 0;
-			String lastDamagePlayer = Lang.legacy("placeholder-no-damage-player");
+			if (hasBoss && (params.startsWith("top_damage") || params.startsWith("top_damager"))) {
+                try {
+					final String[] split = params.split("_");
+                    final int index = split.length < 3 ? 1 : Integer.parseInt(split[2]);
+                    final boolean damager = params.contains("r");
```

Comment:
I suggest to explicitly ask "params.startsWith("top_damager")" here for clarity. The "contains" is typically confusing and can break easily